caddy.go

```diff
@@ -869,7 +873,7 @@ func Stop() error {
 // explicitly like a common local hostname. addr must only
 // be a host or a host:port combination.
 func IsLoopback(addr string) bool {
-	host, _, err := net.SplitHostPort(addr)
+	host, _, err := net.SplitHostPort(strings.ToLower(addr))
```

Comment:
Could you please explain why you changed that?

Reply:
Because I do comparisons against lowercase strings like "localhost"

Reply:
Okay. Thank you!